sphinx_gallery/tests/test_full.py

```diff
@@ -760,3 +760,23 @@ def test_matplotlib_warning_filter(sphinx_app):
     warning = ('Matplotlib is currently using agg, which is a'
                ' non-GUI backend, so cannot show the figure.')
     assert warning not in html
+
+
+def test_jupyter_notebook_pandoc(sphinx_app):
+    """Test using pypandoc."""
+    src_dir = sphinx_app.srcdir
+    fname = op.join(src_dir, 'auto_examples', 'plot_numpy_matplotlib.ipynb')
+    with codecs.open(fname, 'r', 'utf-8') as fid:
+        md = fid.read()
+
+    md_sg = r"Use :mod:`sphinx_gallery` to link to other packages, like\n:mod:`numpy`, :mod:`matplotlib.colors`, and :mod:`matplotlib.pyplot`."  # noqa
+    md_pandoc = r'Use `sphinx_gallery`{.interpreted-text role=\"mod\"} to link to other\npackages, like `numpy`{.interpreted-text role=\"mod\"},\n`matplotlib.colors`{.interpreted-text role=\"mod\"}, and\n`matplotlib.pyplot`{.interpreted-text role=\"mod\"}.'  # noqa
+
+    try:
+        import pypandoc  # noqa
+        # Import error raised only when function called
+        pypandoc.get_pandoc_version()
```

Comment:
Seeing this more than one place makes it seem like we should have a `_get_pandoc()` for example that either returns `pypandoc, version` or `None, None` (DRY)